src/dstore/dstore_base.c

```diff
@@ -59,6 +59,8 @@ int dstore_init(struct collection_item *cfg, int flags)
 
 	assert(dstore && cfg);
 
+	perfc_trace_inii(PFT_DSTORE_INIT, PEM_DSTORE_TO_NFS);
+
```

Comment:
Suggestion: do we need to trace flags?

Reply:
No need because as of now all callers of dstore_init and dsal_init set flag to '0'.

Reply:
ok